archetypes/quickstart/src/brooklyn-sample/pom.xml

```diff
@@ -58,19 +53,30 @@
       <releases>
         <enabled>false</enabled>
       </releases>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
     </repository>
   </repositories>
 
   <dependencies>
     <dependency>
-      <!-- this pulls in all brooklyn entities and clouds; 
-           you can cherry pick selected ones instead (for a smaller build) -->
+      <!-- minimal dependency, for writing entities; consider also brooklyn-software-base -->
       <groupId>org.apache.brooklyn</groupId>
-      <artifactId>brooklyn-all</artifactId>
+      <artifactId>brooklyn-core</artifactId>
```

Comment:
Or instead leave the -all dependency. Now that this doesn't generate a dist it won't make a difference. The `Import-Package` will be generated on the actually used packages.